types/kuzzle_response.go

```diff
@@ -38,7 +38,20 @@ type KuzzleResponse struct {
   Error     MessageError `json:"error"`
 }
 
+type KuzzleSearchResult struct {
+  Hits  []KuzzleResult `json:"hits"`
+  Total int `json:"total"`
+}
+
 type AckResponse struct {
   Acknowledged       bool
   ShardsAcknowledged bool
 }
+
+func (sr KuzzleSearchResult) GetTotal() int {
```

Comment:
Getters are usually used for private members in oop. I think it's a better idea to make exported names, I've rarely seen code written in golang using getters. Let me know what you think!